### ISSUE TITLE
Adding a position options for the PMREM fromScene generation

### ISSUE THREAD

**Issue by Samsy** [mrdoob/three.js#30071]
### Description

Currently, calling a PMREM Generation is calling a cubecamera that is always default positionned at 0, 0, 0

For a few versions now, I'm hacking my way to get to generate a PMREM but from an arbitrary cubecamera position by passing a position to pmrem.fromScene, this would be useful

### Solution

Passing a position as arg to fromScene

### Alternatives

Or maybe passing a callback to be called before each render that passes the camera ?
in that case we'd be able to do anything before render of each pmrem face

**Comment by yuvashrikarunakaran** [comment]
Here’s a potential implementation of fromScene with these enhancements:

fromScene(scene, options = {}) {
  const { position = new THREE.Vector3(0, 0, 0), onBeforeRender } = options;

  // Set up cube camera
  const cubeCamera = new THREE.CubeCamera(near, far, resolution);
  cubeCamera.position.copy(position);

  for (let faceIndex = 0; faceIndex < 6; faceIndex++) {
    if (typeof onBeforeRender === 'function') {
      onBeforeRender(cubeCamera, faceIndex);
    }

    // Render the face
    this.renderer.setRenderTarget(this.targets[faceIndex]);
    this.renderer.render(scene, cubeCamera);
  }

  // Generate PMREM
  return this.pmremGenerator.fromCubeRenderTarget(this.renderer, this.targets);
}

**Comment by Spiri0** [comment]
@Mugen87 can you still remember it? https://github.com/mrdoob/three.js/pull/28334
I tried to solve that back then. This was probably one of my first attempts to understand something in threejs.
But I still had a mistake there. With the extension I tried to make size and position optional and to fix the reflection error that Sunag recently fixed with a PR because that also came up recently. Maybe my approach from back then can be useful

**Comment by Mugen87** [comment]
> Maybe my approach from back then can be useful

I agree. The conclusion of earlier discussion was that having render target size and camera position in control when using `fromScene()` is favorable. Do you think you can create a new version of https://github.com/mrdoob/three.js/pull/28334? #28334 hasn't been merged because of the reflection issue which should be fixed in the meanwhile.

**Comment by Spiri0** [comment]
I've already found time for it and submitted a PR #30334
